plugins/mbeans/src/org/graalvm/visualvm/modules/mbeans/options/GlobalPreferences.java

```diff
@@ -146,12 +146,12 @@ private void addListener(String property, PreferenceChangeListener pcl) {
         }
     }
     
-    private int getPollingInterval(String property, int deflt) {
+    private int getPollingInterval(String property, int default_) {
```

Comment:
This is a little atypical. `default` is a reserved word, but `dflt` could mean `deflate` and so the use of `deflt` isn't a particularly good choice.